### DISCUSSION TITLE
Zed command to path to open editor in current path

### DISCUSSION THREAD

**Discussion by JayMGurav** [zed-industries/zed#6479] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

Provide a feat to add "zed" terminal command to path.

### If applicable, add mockups / screenshots to help present your vision of the feature

<img width="1084" alt="image" src="https://user-images.githubusercontent.com/42145480/224731798-a0f0648e-e12a-4d74-934d-d8de8ff8cf59.png">

**Comment by jansol** [comment]
Like this?

![image](https://user-images.githubusercontent.com/2588851/224732657-f80c3a1f-ba13-49cd-b053-d480a6d7947e.png)

**Comment by JayMGurav** [comment]
Yea sorry, just found out, thanks!

**Comment by jansol** [comment]
I was going to suggest keeping this open for adding it to the in-editor command palette but looks like it is there too already:

<img width="572" alt="image" src="https://user-images.githubusercontent.com/2588851/224733092-18b4da65-e4c1-4df4-a4cb-7e731145913d.png">

**Comment by JayMGurav** [comment]
yea sure, we can have this...should I re-open the issue otherwise??

**Comment by jansol** [comment]
See the above screenshot: we already have it 😆
